functests/test_resources/sbom_story.yaml

```diff
@@ -0,0 +1,109 @@
+---
+# The step field is a string that represents the method bound to an endpoint and
+# the args and kwargs correspond to the arguments to such a method.
+#
+# NB the assets and events endpoints require all values to be strings. Other values may
+# be of the correct type such as confirm which is a boolean.
+#
+# This story creates a sbom and executes release events on that SBOM.
+# 
+# sbom assets are signified by arc_display_type=SBOM
+# sbom releases are signified by arc_display_type=SBOM_RELEASE
+#
+steps:
+  - step:
+      action: ASSETS_CREATE_IF_NOT_EXISTS
+      description: Create the ACME Corporation Detector SAAS SBOM
+      asset_label: ACME Corporation Detector SAAS
+    selector:
+      - attributes:
+        - arc_display_name
+        - arc_namespace
+    behaviours:
+      - RecordEvidence
+      - Attachments
+    attributes:
+      arc_display_name: ACME Corporation Detector SAAS
+      arc_namespace: !ENV ${ARCHIVIST_NAMESPACE:namespace}
+      arc_display_type: SBOM
```

Comment:
We use "Software Package" as the arc_display_type in the other examples. This is more than just an SBOM - it's the complete release history of a package.

Reply:
fixed - also the events now have a type of "Software Package Release" to be consistent